### PR TITLE
QF Crossbar log & kz_network_utils:verify_cidr/2

### PR DESCRIPTION
**Pull request by skorobkov**
Fix kz_network_utils:verify_cidr/2 (it never match)
Display IPv4 addres when it is mapped in IPv6 (0:0:0:0:0:ffff:0:0/96 [RFC](https://tools.ietf.org/html/rfc4038#section-4.2))

### CODE REVIEW

File: core/kazoo/src/kz_network_utils.erl [hunk 1]
```diff
@@ -151,7 +152,8 @@ verify_cidr(IP, CIDR) when is_binary(CIDR) ->
     verify_cidr(IP, kz_term:to_list(CIDR));
 verify_cidr(IP, CIDR) ->
     Block = inet_cidr:parse(CIDR),
-    inet_cidr:contains(Block, inet:parse_address(IP)).
+    {'ok', IPTuple} = inet:parse_address(IP),
```
Comment (skorobkov) [inline core/kazoo/src/kz_network_utils.erl]: I thought about it too.
It minimize changes, then the simpliest way is change [kz_network_utils:iptuple_to_binary/1](https://github.com/2600hz/kazoo/blob/master/core/kazoo/src/kz_network_utils.erl#L309) then it always return IPv4 address when it found 0:0:0:0:0:ffff:0:0/96 address.

**Comment by skorobkov** [comment]
I thought about it too.
It minimize changes, then the simpliest way is change [kz_network_utils:iptuple_to_binary/1](https://github.com/2600hz/kazoo/blob/master/core/kazoo/src/kz_network_utils.erl#L309) then it always return IPv4 address when it found 0:0:0:0:0:ffff:0:0/96 address.

**Comment by skorobkov** [comment]
@jamhed ???

**Comment by jamhed** [comment]
@skorobkov, nm, making test suite work.

**Comment by lazedo** [comment]
@jamhed lets try mkbusy tests with other pr (maybe use your personal repo) , merging this.

**Pull request merged**